open_academy/views/open_academy_partner_views.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please rename this file. Should be `res_partner_views.xml`

![image](https://user-images.githubusercontent.com/11479473/182206968-2bc77453-6e02-4a7b-b1c5-3c140606c010.png)